os/manual-rollbacks.md

```diff
@@ -2,6 +2,8 @@
 
 As much as we all love flawless, automatic updates, there may be occasions when an update must be rolled back. This is fairly straightforward on Container Linux, once you know the right commands.
 
+**Note**: Container Linux allows you to rollback to a previously active partition, or change to an older release channel (to downgrade to a stable version). It does not allow you to downgrade to an uninstalled, user defined version number.
```

Comment:
I think the note goes right at the top. People hitting this page should know that there are limited rollback options. And I think the note was added specifically because users were calling tech support, complaining that rolling back to a defined version number wasn't working as expected.

question - can be rolled back to a version installed immediately after an upgrade? or can be rolled back, immediately after an upgrade, to the version installed on the rollback partition?

The active version of Container Linux can be rolled back to the version installed on the rollback partition immediately after an upgrade, or downgraded to the version current on any lower release channel. Rolling back to an arbitrary previous version number is not supported.